### PR TITLE
Add initial api spec

### PR DESCRIPTION
**Pull request by tonistuulNG**

### CODE REVIEW

File: docs/technical/api-spec.yml [hunk 1]
```diff
@@ -0,0 +1,327 @@
...
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/Participant'
+      description: ''
```
Comment (turnerrainer) [inline docs/technical/api-spec.yml]: Please add description. Is it to show the list of all participants?

File: docs/technical/api-spec.yml [hunk 2]
```diff
@@ -0,0 +1,327 @@
+openapi: 3.0.3
+info:
+  title: Central Operations API
+  version: '1.0'
```
Comment (turnerrainer) [inline docs/technical/api-spec.yml]: Definitely not version `1.0` but `0.1`

File: docs/technical/api-spec.yml [hunk 3]
```diff
@@ -0,0 +1,327 @@
...
+      parameters:
+        - schema:
+            type: string
+          in: query
+          name: type
+        - schema:
+            type: string
+          in: query
+          name: institutionId
+        - $ref: '#/components/parameters/X-Api-Key'
```
Comment (turnerrainer) [inline docs/technical/api-spec.yml:214]: Disturbing but fine for initial setup

File: docs/technical/api-spec.yml [hunk 4]
```diff
@@ -0,0 +1,327 @@
...
+          description: OK
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/PublicInstitution'
+      operationId: get-public-api-institutions
+      parameters:
+        - $ref: '#/components/parameters/X-Api-Key'
```
Comment (turnerrainer) [inline docs/technical/api-spec.yml:233]: Disturbing but fine for initial setup

**Review by turnerrainer — CHANGES_REQUESTED** [review]
Some minor changes needed

**Review by joonasroosalung — APPROVED** [review]

**Review by turnerrainer — APPROVED** [review]

**Pull request merged**